Gaijinizer patch/CommonEvents_597.txt

```diff
@@ -4,72 +4,69 @@
 
 うぅ……
 > CONTEXT: CommonEvents_597/7/Dialogue/kazuya-0 
-Hah... haaah......
+Haah...... Haah......
 
-Nnnn...#Triketos
+Uugh......#Triketos/Strate
 > END STRING
 
 > BEGIN STRING
 ハーピーモドキは、胸や顔についた精液を舐めとりながら
 
 こちらを面白そうに見つめている……
 > CONTEXT: CommonEvents_597/13/Dialogue 
-The Beetle Harpy licks up the semen on her
-chest and face as she stares deep into my eyes
-with an amused look...#Triketos
+The Harpy Spawn stares at me in amusement as she licks the
+semen on her chest and face......#Triketos/Strate
 > END STRING
 
 > BEGIN STRING
 『そうか、お前はこういうのが好きなのか？』
 
 とでも言わんばかりの、蔑みの含まれた目つきだ……
 > CONTEXT: CommonEvents_597/17/Dialogue 
-It's as if she's saying, "Oh, so you
-liked that?" as she looks down on me...#Triketos
+It's as if she were saying "So you like this kind of thing,
+huh?" with that look of disdain on her face......#Triketos/Strate
 > END STRING
 
 > BEGIN STRING
 うぁぁぁ……っ！♡♡
 > CONTEXT: CommonEvents_597/42/Dialogue/kazuya-0 
-W-wah...!♡♡#Triketos
+Uaaah......! ♡♡#Triketos/Strate
 > END STRING
 
 > BEGIN STRING
 ギャッ♡ギャッ♡
 > CONTEXT: CommonEvents_597/45/Dialogue/enemy21 
 > CONTEXT: Map296/events/55/pages/0/54/Dialogue/enemy21 
-Gya~tsu♡ gya~tsu♡#GT
+Gyaa♡ Gyaa♡#Strate
 > END STRING
 
 > BEGIN STRING
 ハーピーモドキは、「もっとしてやろう」と言うかのように
 
 さっきよりも激しく、胸でペニスをもみくちゃにした……！
 > CONTEXT: CommonEvents_597/47/Dialogue 
-She then starts intensely squeezing my penis
-between her breasts, in addition to her tongue
-as if she was saying, 
-"Let me do some more..."#Triketos
+As if to say "I'll do even more", she starts crushing my
+penis between her breasts even more intensely than
+before......!#Triketos/StrateUnsure
 > END STRING
```

Comment:
Note

Reply:
Translation looks fine to me.

Reply:
Okay, then... "I'll do even more" -> "I'll do it even more".